vnext/Shared/JSI/V8RuntimeHolder.cpp

```diff
@@ -0,0 +1,263 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "V8RuntimeHolder.h"
+#include <ApiLoaders/V8Api.h>
+#include <NodeApiJsiRuntime.h>
+#include <crash/verifyElseCrash.h>
+
+using namespace Microsoft::NodeApiJsi;
+
+#define CRASH_ON_ERROR(result) VerifyElseCrash(result == napi_ok);
+
+namespace Microsoft::ReactNative {
+namespace {
+
+class V8FuncResolver : public IFuncResolver {
+ public:
+  // TODO: (vmoroz) Use Office safe DLL loading
```

Comment:
Can we either do this, or file an issue to follow up on it?

Reply:
Good catch! I will do it before merging.